src/daemon/__DAEMON_PACKAGES__

```diff
@@ -1,7 +1,13 @@
 \
-          sharutils \
+          ca-certificates \
           __CONFD_PACKAGE__ \
+          e2fsprogs \
+          __ETCD_PACKAGE__ \
           __FOREGO_PACKAGE__ \
+          __GANESHA_PACKAGES__ \
+          __ISCSI_PACKAGES__ \
+          kmod \
           __KUBECTL_PACKAGE__ \
-          __ETCD_PACKAGE__ \
-          __S3CMD_PACKAGE__
+          lvm2 \
```

Comment:
`lvm2` is needed by `ceph-volume`.

Reply:
If it is needed, it is installed by `yum`, as it is a tracked dependency.

Reply:
I don't think it is. @ktdreyer or is it?
If we have that here, I guess this is for a good reason :)

Reply:
+1

Reply:
ceph-osd ask for lvm2. So that's safe.